syntax = "proto3";

service ShowMessage {
    rpc show(InputMessage) returns (OutputMessage) {}
}

message InputMessage {
    string value = 1;
}

message OutputMessage {
    string value = 1;
}

service RecognizeMessage {
    rpc recognize(InputMessage) returns (OutputMessage) {}
}